packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -705,16 +716,19 @@ export default class SalesforceCRMService implements CRM {
     attendee,
     recordType,
     organizerId,
+    calEventResponses,
   }: {
     attendee: { email: string; name: string };
     recordType: SalesforceRecordEnum;
     organizerId?: string;
+    /**Only Leads have the default company field */
+    calEventResponses?: CalEventResponses | null;
   }) {
     const [FirstName, LastName] = attendee.name ? attendee.name.split(" ") : [attendee.email, ""];
 
     // Assume that the first part of the email domain is the company title
-    const company = attendee.email.split("@")[1].split(".")[0];
-
+    const company =
```

Comment:
This is the biggest change since we pass the company value when the lead is created. So we need to read the booking responses now vs when the event is created.